services/discovery/v1.js

```diff
@@ -30,6 +31,25 @@ module.exports = function (RED) {
     endpoint = '',
     sEndpoint = 'https://gateway.watsonplatform.net/discovery/api';
 
+const ExecutionList = {
+  'createEnvrionment': executeCreateEnvrionment,
+  'listEnvrionments': executeListEnvrionments,
+  'getEnvironmentDetails': executeEnvrionmentDetails,
+  'createCollection': executeCreateCollection,
+  'listCollections': executeListCollections,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['executeListCollections' was used before it was defined.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4846228)